### DISCUSSION TITLE
Statistical analysis of ancestry adjusted output

### DISCUSSION THREAD

**Discussion by SalemWerdyani** [PGScatalog/pgsc_calc#388] [Q&A] [answered]
Hi @smlmbrt, Thank you for the tool and pipeline, since pigs_calc provide the SUM and three ansectry correction including Z_MostSimilarPop, Z_norm1, and Z_norm2, which of these scores is the most accurate to use in the further analyses, and do these ancestry normalization control for PGS confounders including the population stratification, or do we still need to use the top 10 PCs in the statistical analyses. Thank you

**Comment by smlmbrt** [comment, marked as answer]
@SalemWerdyani, to some extent this will depend on your research question and your populations. You can usually test which is the best performing normalization and whether any is more calibrated (overall they're quite correlated). With respect to including PCs, this also depends on your research question, including PCs in a regression is trying to adjust for the relationships between PCs and the outcome, not the genetic predisposition (estimated by the PGS).
